cmd/config/virt-udn-density/Containerfile

```diff
@@ -0,0 +1,6 @@
+FROM registry.access.redhat.com/ubi8/ubi:latest AS builder
```

Comment:
I don't think this Containerfile is needed, I think Raul originally had this in PR#154 because he was building his own VM cirros image, it makes sense to standardize on the `quay.io/openshift-cnv/qe-cnv-tests-fedora:40` images you use in this PR.